### PR TITLE
Rename bard facts model

### PR DESCRIPTION
**Pull request by balajimanoharan**
**BREAKING CHANGE**
-  Rename the bard-facts model to something more appropriate 
- Move the fetchNext and fetchPrevious methods from model to service

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kevinhinterlong** [comment]
Rename to bard to fili as well?

File: packages/data/addon/models/navi-facts.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2017, Yahoo Holdings Inc.
```
Comment (mattgraham1995) [inline packages/data/addon/models/navi-facts.js]: 2019

File: packages/data/addon/models/navi-facts.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Description: A model that holds the response from a Bard query.
+ */
+
+import EmberObject from '@ember/object';
+
+export default EmberObject.extend({
```
Comment (jkusa) [inline packages/data/addon/models/navi-facts.js]: Can this be an es6 class?

**Comment by balajimanoharan** [comment]
> Rename bard-facts to fili-facts?

@kevinhinterlong the idea is to use the same model for any datasource.`navi-facts` is therefore apt for this reason.

**Comment by kevinhinterlong** [comment]
> > Rename bard-facts to fili-facts?
> 
> @kevinhinterlong the idea is to use the same model for any datasource.`navi-facts` is therefore apt for this reason.

Yea, just pointing out that the linked ticket says

> We have some services named `bard-facts`, `bard-dimensions`, `bard-metadata`. Bard has been renamed to fili, so this could also be renamed appropriately.

so I wasn't sure if the rename of bard -> fili was missed

**Pull request merged**